interface/src/app/gene/searchMany/search-buttonset.tpl.html

```diff
@@ -1,7 +1,18 @@
 <div class="btn-group">
-    <no-result-button query="{{query}}"></no-result-button>
-    <previous-result-button ng-if="previousPages"></previous-result-button>
-    <gene-result-button query="{{query}}" ng-repeat="gene in pageGenes">
-    </gene-result-button>
-    <more-result-button ng-if="additionalPages"></more-result-button> 
+  <button class="btn btn-default" ng-click="removeGene()">
+    <i class="fa fa-minus-square"></i> None
+  </button>
+  <button class="btn btn-default" ng-if="previousButtonPages"
+    ng-click="previousGenePage()">
+      <i class="fa fa-angle-double-left"></i> Previous
+  </button>
+
+  <gene-result-button query="{{query}}" search-results="searchResults"
+    queries="queries" ng-repeat="gene in buttonPageGenes" gene="gene">
+  </gene-result-button>
+
+  <button class="btn btn-default" ng-click="nextGenePage()"
+    ng-if="additionalButtonPages">
+      More <i class="fa fa-angle-double-right"></i>
+  </button>
```

Comment:
I take it all of these buttons always appear together as a button set. If so, I think this strikes a great balance between reusable module and the "granularity" of re-use.